doc/concepts/leader_election.md

```diff
@@ -0,0 +1,38 @@
+### Leading ETCD main container’s sidecar is the backup leader 
+
+- The `backup-restore sidecar` poll its own etcd main container to see if it is the leading member in the etcd cluster.
+This information is used by the backup-restore sidecars to decide that sidecar of the leading etcd main container is the `backup leader`.
```

Comment:
Consider replacing with the following
`The backup-restore sidecar corresponding to the leading etcd member becomes the backup leader while the other backup-restore sidecars become followers`

Reply:
here I'm just wanted to explain the about `backup leader` only.

Reply:
Then maybe consider only the first part of the sentence